tf_agents/utils/nest_utils_test.py

```diff
@@ -680,6 +680,20 @@ def testWhere(self):
     expected = (np.array([0, 1, 1, 0, 1]), np.array([1, 7, 8, 4, 10]))
     self.assertAllEqual(expected, result)
 
+  def testWhereV2(self):
```

Comment:
can you test with empty tensors to ensure the right error message etc?

Reply:
Sure. Let me think about what tests make the most sense here and I will update.

Reply:
Added tests for different ranks and same rank different dimension tensore. Existing test for `nest_utils.where` only covered same rank/same dimension.

Reply:
great, thanks!